be/src/vec/sink/group_commit_block_sink.cpp

```diff
@@ -200,9 +200,31 @@ Status GroupCommitBlockSink::_add_blocks() {
     load_id.__set_lo(_load_id.lo);
     if (_load_block_queue == nullptr) {
         if (_state->exec_env()->wal_mgr()->is_running()) {
+            std::string wal_dir;
+            auto* wal_mgr = ExecEnv::GetInstance()->wal_mgr();
+            {
+                std::unique_lock l(lock);
+                auto map = wal_mgr->get_wal_dir_to_disk_usage_map();
+                wal_dir = map.size() == 1
```

Comment:
Abstract the select wal_dir to a method.
Is it better to move this method into the while loop?